dev/openchpl_views.sql

```diff
@@ -1378,3 +1380,23 @@ FROM openchpl.certified_product cp
          AND cse.certification_status_id IN (1, 6, 7)) as listing_status
          ON cp.certified_product_id = listing_status.certified_product_id
 AND cp.certification_edition_id = 3;
+
+CREATE OR REPLACE VIEW openchpl.requirement_type
```

Comment:
I see that these views already exist in the database and code references them. Are you just putting them here because we missed them at some point? Are there any changes to them?